src/Hooks/GutenbergCleanup.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace WPML\PB\Elementor\Hooks;
+
+use WPML\FP\Fns;
+use WPML\FP\Maybe;
+use WPML\FP\Obj;
+use WPML\FP\Relation;
+use WPML\PB\Elementor\DataConvert;
+use WPML_Elementor_Data_Settings;
+use function WPML\FP\partial;
+use function WPML\FP\pipe;
+
+class GutenbergCleanup implements \IWPML_Backend_Action, \IWPML_Frontend_Action, \IWPML_DIC_Action {
+
+	public function add_hooks() {
+		add_filter(
+			'update_post_metadata',
+			Fns::withoutRecursion( Fns::identity(), [ $this, 'removeGutenbergFootprint' ] ),
```

Comment:
@strategio can you explain `Fns::identity()` ?

I've always used `Fns::noop()` there

Reply:
@dgwatkins, if we are in a recursion `Fns::withoutRecursion` will call the first function, `Fns::identify` here, so it will return the original value passed to the filter.

At some point, it's like replacing our current filter with a filter that just returns the value it receives.

If I used `Fns::noop`, it would be a filter that always returns `null` (because does not return anything), regardless of the input value.

Did I missed something?

Reply:
Makes sense, thanks!